tests/swarm/test_bandwidth_governor.py

```
@@ -0,0 +1,460 @@
+        agents = [create_agent_id(f"SAT{i:03d}") for i in range(10)]
+        
+        # Each should get ~1KB/s fair share
+        for agent in agents:
+            governor._get_peer_bucket(agent)
+        
+        fair_share = governor.fair_share_per_peer()
+        assert fair_share == pytest.approx(1000.0, rel=0.01)
+    
+    def test_burst_traffic_critical_priority(self):
+        """Test 2x burst traffic with CRITICAL priority."""
+        config, agent_id = create_config()
```

Ruff (0.14.10)

Unpacked variable `agent_id` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)